.github/workflows/long-running-azure.yaml

```diff
@@ -78,6 +78,9 @@ env:
   # The valid radius build time window in seconds to rebuild radius. 24 hours = 24 * 60 * 60 = 86400
   VALID_RADIUS_BUILD_WINDOW: 86400
 
+  # The functional test GitHub app id
+  FUNCTIONAL_TEST_APP_ID: 425843
```

Comment:
Should this be a repo secret or env var?

Reply:
It's not a secret in our functional tests but will double check with a maintainer